chart/openfaas/README.md

```diff
@@ -87,6 +87,27 @@ By default services will be exposed with following hostnames (can be changed, se
 * `prometheus.openfaas.local`
 * `alertmanager.openfaas.local`
 
+### Secure the Gateway administrative API and UI with basic auth
+
+In order to enable basic auth first you need to create a secret named `basic-auth` in the `openfaas` namespace:
+
+```bash
+# generate a random password
+password=$(head -c 12 /dev/random | shasum| cut -d' ' -f1)
```

Comment:
I found that this hangs on DigitalOcean, using `/dev/urandom` is faster and should work regardless without blocking.